.github/workflows/automate-sample-apps.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger a repository dispatch event to sample apps repo to run apps
```

Comment:
As discussed offline, remove this file. We do not want to use personal access tokens (even as secrets).